acceptance/acceptance_test.go

```diff
@@ -186,8 +187,22 @@ func accept(t *testing.T, params acceptParms) {
 		".",
 	)
 	cmd.Dir = "./testdata"
+	stdout, err := cmd.StdoutPipe()
+	require.NoError(t, err)
+	stderr, err := cmd.StderrPipe()
+	require.NoError(t, err)
 	t.Log("will exec:", cmd.Args)
-	bts, err := cmd.CombinedOutput()
-	t.Log("output:", string(bts))
+	err = cmd.Start()
+
+	slurp, _ := ioutil.ReadAll(stderr)
+	if len(slurp) > 0 {
+		t.Log("stderr:", string(slurp))
```

Comment:
should we assert this with something?

Reply:
No because data on the subprocess stderr is not always an error in the call